tests/conftest.py

```diff
@@ -31,8 +32,14 @@
     "transformers_llama3cpu_8b": dict(
         # Note that this model requires an appropriate
         # HF_TOKEN environment variable
-        name="meta-llama/Meta-Llama-3-8B-Instruct",
-        kwargs={"trust_remote_code": True},
+        name="transformers:meta-llama/Meta-Llama-3-8B-Instruct",
+        kwargs={"trust_remote_code": True, "torch_dtype": torch.bfloat16},
```

Comment:
@riedgar-ms is this necessary? it seems the config.json for these models already specifies bf16 - this makes all tests require torch

Reply:
I can experiment with removing it; I was simply copying the initialisation from the model card. The `test` 'extra' for Guidance was already installing `torch`.